### PR TITLE
Add workflow definition for fireperf end-to-end tests

### PR DESCRIPTION
**Pull request by yifanyang**

### CODE REVIEW

**Comment by yifanyang** [comment]
@vkryachko A successful execution of this workflow can be found at https://github.com/firebase/firebase-android-sdk/actions/runs/2800178050.

**Review by vkryachko — COMMENTED** [review]
LGTM overall.

It's unclear to me how notifications will work though, [afaict](https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/notifications-for-workflow-runs) you will be the one getting failure notifications, not the fireperf team. wdyt?

**Comment by yifanyang** [comment]
> LGTM overall.
> 
> It's unclear to me how notifications will work though, [afaict](https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/notifications-for-workflow-runs) you will be the one getting failure notifications, not the fireperf team. wdyt?

Added the last step that create/update issue to notify fireperf folks. Here is an example issue #3977. 

I have removed the `pull_request` trigger, therefore the tests did not run for last commit. Successful runs can be found at: 
- https://github.com/firebase/firebase-android-sdk/runs/7757358619?check_suite_focus=true
- https://github.com/firebase/firebase-android-sdk/runs/7757358696?check_suite_focus=true

File: .github/workflows/fireperf-e2e.yml [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+                body: text,
+              });
+            } else {
+              github.rest.issues.create({
+                owner: owner,
+                repo: repo,
+                title: 'FirePerf E2E Test Failures',
+                body: text,
+                labels: ['fireperf-e2e-tests'],
+                assignees: ['jeremyjiang-dev', 'leotianlizhan', 'visumickey']
```
Comment (vkryachko) [inline .github/workflows/fireperf-e2e.yml:105]: fyi @jeremyjiang-dev @leotianlizhan @visumickey does that look good to you?

Comment (jeremyjiang-dev) [reply .github/workflows/fireperf-e2e.yml:105]: Could you also add @raymondlam ? LGTM

Comment (yifanyang) [reply .github/workflows/fireperf-e2e.yml]: Done.

**Review by vkryachko — APPROVED** [review]

**Pull request merged**